pkg/scheduler/algorithm/priorities/resource_allocation.go

```diff
@@ -91,27 +93,38 @@ func (r *ResourceAllocationPriority) PriorityMap(
 	}, nil
 }
 
-// getNonZeroRequests returns the total non-zero requests. If Overhead is defined for the pod and the
-// PodOverhead feature is enabled, the Overhead is added to the result.
-func getNonZeroRequests(pod *v1.Pod) *schedulernodeinfo.Resource {
-	result := &schedulernodeinfo.Resource{}
-	for i := range pod.Spec.Containers {
-		container := &pod.Spec.Containers[i]
-		cpu, memory := priorityutil.GetNonzeroRequests(&container.Resources.Requests)
-		result.MilliCPU += cpu
-		result.Memory += memory
-	}
-
-	// If Overhead is being utilized, add to the total requests for the pod
-	if pod.Spec.Overhead != nil && utilfeature.DefaultFeatureGate.Enabled(features.PodOverhead) {
```

Comment:
we need this logic to be reflected in calculatePodResourceRequest

Reply:
added it 👍